Kernel/Storage/PATADiskDevice.h

```diff
@@ -80,6 +80,7 @@ class PATADiskDevice final : public StorageDevice {
     u16 m_cylinders { 0 };
     u16 m_heads { 0 };
     u16 m_sectors_per_track { 0 };
+    u64 m_max_lba { 0 };
```

Comment:
Instead of creating a new field in `PATADiskDevice`, you could pass the max LBA value for the `StorageDevice` constructor.